GUI/DeusEx/Classes/DXRMenuUIWindow.uc

```diff
@@ -0,0 +1,7 @@
+class DXRMenuUIWindow shims MenuUIWindow;
```

Comment:
you're making this change globally? should just be inside `GUI\DeusEx\Classes\DXRMenuScreenNewGame.uc`

Reply:
I think it's better for the shortcut to be consistent everywhere.

Reply:
did you check if anything uses alt+e?

Reply:
I looked through all the settings windows, but some grepping reveals I missed a couple windows. (Amusingly, the Display window already has two alt+e shortcuts.)

Reply:
I'm just going to remove the shortcut entirely from the New Game window rather than give it one that's different from everywhere else. It's not like anyone's using it on purpose anyway.